docs/nx-cloud/private/ami-setup.md

```diff
@@ -14,24 +14,24 @@
    - Allow SSH from your current IP
 6. Leave the storage options as they are
 7. "Launch instance"
-8. Wait 10 minutes, then navigate to your instance's IP in the browser. You should see the NxCloud dashboard!
+8. Wait 10 minutes, then navigate to your instance's IP in the browser. You should see the Nx Cloud dashboard!
 
-![NxCloud landing page](/nx-cloud/private/images/nx-cloud-landing.png)
+![Nx Cloud landing page](/nx-cloud/private/images/nx-cloud-landing.png)
 
-### Your NxCloud URL
+### Your Nx Cloud URL
 
 1. At this point, your instance will have a public IP accessible from the browser.
-   - You can consider this IP the URL of NxCloud, and proceed with the below steps and all will work fine!
+   - You can consider this IP the URL of Nx Cloud, and proceed with the below steps and all will work fine!
 2. You might want, however, to add a Load Balancer in front of the instance, with an explicit domain (e.g. https://my-nxcloud.my-org.com).
-   - This is strongly recommended because you will be able to upgrade/restart/re-configure your NxCloud EC2 instance while keeping the NxCloud URL static.
+   - This is strongly recommended because you will be able to upgrade/restart/re-configure your N xCloud EC2 instance while keeping the Nx Cloud URL static.
```

Comment:
```suggestion
   - This is strongly recommended because you will be able to upgrade/restart/re-configure your Nx Cloud EC2 instance while keeping the Nx Cloud URL static.
```

Reply:
Fixed. Thanks!